src/main/java/com/revature/services/PersonLogicImpl.java

```diff
@@ -72,9 +72,9 @@ public Person getPersonById(int id)  {
 			p = dao.getOne(id);
 			System.out.println("This is my P: "+p);
 			 
-		 } catch (Exception e) {		
+		 } catch (EntityNotFoundException e) {		
 			 
-			 p = new Person("","",0);
+			 p = new Person("Persondoes","Notexist",0);
 			 p.setId(0);
 			 System.out.println("setting P to new person");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)